### PR TITLE
Renamed LF to HF

### PR DESCRIPTION
**Pull request by MinickLS**

### CODE REVIEW

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_HF.js [hunk 1]
```diff
@@ -1,59 +1,59 @@
...
-};
-   
-module.exports = ReaderArduino_RC522_LF;
+
+var bluetooth = require('./CommBluetooth');
+var resources = require('./Resources');
+var ReaderArduino_RC522_HF = {};
+
+/**
+ * Provides a parser to parse the RC522 LF Bluetooth device. This calls
```
Comment (kaysabelle) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_HF.js]: Forgot one here

**Comment by jeslin** [comment]
RIP Fake LF reader :skull: lolol looks good to me!

edit: JK GOOD CALL KATRINA

File: radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_HF.js (hunk 1 above)
Comment (jeslin) [inline radlibDemoApp/plugins/com.radlib.cordova.rfidreader/www/ReaderArduino_RC522_HF.js]: and one here

**Comment by kaysabelle** [comment]
You forgot one in a comment but otherwise looks good!

**Pull request merged**